src/components/modals/ProfileModal.vue

```diff
@@ -1,33 +1,41 @@
 <template>
-  <div v-if="user" class="overlay" @click.self="close">
-    <div class="modal__main--flex profileModal__centered">
+  <modal-wrapper v-if="user" @close="close">
```

Comment:
Bør denne nå ta i bruk header fra `ModalWrapper` for å "aligne" enda mer? I samme åndedrag også fjerne margin for `.column` (og ev. senere eventuelt introdusere mer padding i modalene globalt?)

<img width="572" alt="image" src="https://user-images.githubusercontent.com/2866225/195322194-797dcc4c-8f08-4294-9cf0-16c41b61a14f.png">

Ser forøvrig vi nå bør ha en sjekk på hvorvidt footer-sloten brukes (i `ModalWraper`), i og med at den ikke brukes her, for å unngå at footer-wrapperen tar opp (uproporsjonalt mye) plass.

<img width="537" alt="image" src="https://user-images.githubusercontent.com/2866225/195323397-ec8e7b57-ec0e-42e8-b3f4-443c35da46c1.png">

Reply:
> Bør denne nå ta i bruk header fra `ModalWrapper` for å "aligne" enda mer?

Jepp.

> Ser forøvrig vi nå bør ha en sjekk på hvorvidt footer-sloten brukes (i `ModalWraper`), i og med at den ikke brukes her, for å unngå at footer-wrapperen tar opp (uproporsjonalt mye) plass.

Ja, jeg tenkte at vi heller burde redesigne hele den modalen der senere til å ta i bruk footeren og ligne mer på de andre. Litt rart å plutselig ha lagreknappen på midten der..?

Reply:
> Ja, jeg tenkte at vi heller burde redesigne hele den modalen der senere til å ta i bruk footeren og ligne mer på de andre. Litt rart å plutselig ha lagreknappen på midten der..?

Enig. (Uansett kan man tenke seg mulige fremtidige modaler uten footer, men er for så vidt ikke relatert til denne PR-en.)